pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [project]
 name = 'mmda'
-version = '0.9.9'
+version = '0.10.0'
```

Comment:
I swear I'm always confused by semver so i might be wrong again but shouldn't this be 0.9.10 (only increase the last number because it is a minor change that doesn't really affect the whole MMDA library as a whole?)

Reply:
i guess I considered it a breaking change, but I think you're right since no one actually depends on the specific behavior that's changing in this case.

Reply:
yeah, I think we usually consider the ai2 model updates as minor